type.proto

```diff
@@ -607,8 +605,41 @@ message PtObject {
     optional int32 score = 19; // DEPRECATED
     optional Trip trip = 20;
     repeated int32 scores = 21;
+    repeated EquipmentDetails equipment_details = 22;
```

Comment:
It should be in `StopPoint`

Reply:
From afar, transfers (and maybe vehicles if we consider PRM equipment) can have some equipment too.

Reply:
As discussed in refinement, it is in the section 'from' and 'to', at the same level as the stop_point.
Check the mock response in https://jira.kisio.org/browse/NAVP-1210.

Reply:
That's not the scope of this PR.